### ISSUE TITLE
[BUG]Percy/Agent is calling for chromium but arm64 version not available cause build failure

### ISSUE THREAD

**Issue by peterzhuamazon** [opensearch-project/OpenSearch-Dashboards#996]
**Describe the bug**

```

[5/5] Building fresh packages...
[11/46] ⡀ puppeteer
[10/46] ⡀ yo
[8/46] ⡀ re2
[9/46] ⡀ lmdb-store
error /tmp/tmpfzrxx026/OpenSearch-Dashboards/node_modules/puppeteer: Command failed.
Exit code: 1
Command: node install.js
Arguments:
Directory: /tmp/tmpfzrxx026/OpenSearch-Dashboards/node_modules/puppeteer
Output:
The chromium binary is not available for arm64:
If you are on Ubuntu, you can install with:

 apt-get install chromium-browser

/tmp/tmpfzrxx026/OpenSearch-Dashboards/node_modules/puppeteer/lib/cjs/puppeteer/node/BrowserFetcher.js:112
            throw new Error();
            ^





ERROR [bootstrap] failed:
ERROR Error: Command failed with exit code 1: /home/ec2-user/.nvm/versions/node/v10.24.1/lib/node_modules/yarn/bin/yarn.js install --non-interactive
          at makeError (/tmp/tmpfzrxx026/OpenSearch-Dashboards/packages/osd-pm/dist/index.js:25139:11)
          at handlePromise (/tmp/tmpfzrxx026/OpenSearch-Dashboards/packages/osd-pm/dist/index.js:24074:26)
          at process._tickCallback (internal/process/next_tick.js:68:7)
error Command failed with exit code 1.
info Visit https://yarnpkg.com/en/docs/cli/run for documentation about this command.
2021-12-06 19:04:18 ERROR    Error building OpenSearch-Dashboards, retry with: ./build.sh manifests/1.3.0/opensearch-dashboards-1.3.0.yml --component OpenSearch-Dashboards
Traceback (most recent call last):
  File "./src/run_build.py", line 83, in <module>
    sys.exit(main())
  File "./src/run_build.py", line 71, in main
    builder.build(build_recorder)
  File "/home/ec2-user/opensearch-build/src/build_workflow/builder_from_source.py", line 49, in build
    self.git_repo.execute(build_command)
  File "/home/ec2-user/opensearch-build/src/git/git_repository.py", line 83, in execute
    subprocess.check_call(command, cwd=cwd, shell=True)
  File "/usr/lib64/python3.7/subprocess.py", line 363, in check_call
    raise CalledProcessError(retcode, cmd)
subprocess.CalledProcessError: Command 'bash /home/ec2-user/opensearch-build/scripts/components/OpenSearch-Dashboards/build.sh -v 1.3.0 -p linux -a arm64 -s false -o builds' returned non-zero exit status 1.
```

You can also reproduce this by cloning opensearch-dashboards then `yarn osd bootstrap` on a arm64 host.
You can temp resolve this by remove percy/agent from package.json.

**Expected behavior**
Due to chromium binary not exist for arm64 it is not able to build dashboards correctly on arm64 hosts.
This step should complete with no issues.

**OpenSearch Version**
Please list the version of OpenSearch being used.

**Dashboards Version**
Please list the version of OpenSearch Dashboards being used.

**Plugins**

Please list all plugins currently enabled.

**Screenshots**

**Host/Environment (please complete the following information):**
 - OS: [LINUX]
 - Browser and version [arm64, al2]

**Additional context**

**Comment by peterzhuamazon** [comment]
After removing percy/agent, the yarn bootstrap is running but building process is failed:
```

   │ERROR webpack compile errors
       │ERROR [core] build
           │ERROR Optimizations failure.
           │         332 modules
           │
           │          ERROR in ./public/index.scss?v7dark (/tmp/tmpydnnjqyv/OpenSearch-Dashboards/node_modules/css-loader/dist/cjs.js??ref--6-oneOf-0-1!/tmp/tmpydnnjqyv/OpenSearch-Dashboards/node_modules/postcss-loader/src??ref--6-oneOf-0-2!/tmp/tmpydnnjqyv/OpenSearch-Dashboards/node_modules/sass-loader/dist/cjs.js??ref--6-oneOf-0-3!./public/index.scss?v7dark)
           │          Module build failed (from /tmp/tmpydnnjqyv/OpenSearch-Dashboards/node_modules/sass-loader/dist/cjs.js):
           │          SassError: Top-level selectors may not contain the parent selector "&".
           │                  on line 1:10 of src/core/public/chrome/ui/header/_index.scss
           │                  from line 1:9 of src/core/public/chrome/ui/_index.scss
           │                  from line 1:9 of src/core/public/chrome/_index.scss
           │                  from line 5:9 of /tmp/tmpydnnjqyv/OpenSearch-Dashboards/src/core/public/index.scss
           │          >>   &.euiBody--headerIsFixed {
           │
           │             --^
           │
           │          SassError: SassError: Top-level selectors may not contain the parent selector "&".
           │                  on line 1:10 of src/core/public/chrome/ui/header/_index.scss
           │                  from line 1:9 of src/core/public/chrome/ui/_index.scss
           │                  from line 1:9 of src/core/public/chrome/_index.scss
           │                  from line 5:9 of /tmp/tmpydnnjqyv/OpenSearch-Dashboards/src/core/public/index.scss
           │          >>   &.euiBody--headerIsFixed {
           │
           │             --^
           │
           │              at Object.render [as callback] (/tmp/tmpydnnjqyv/OpenSearch-Dashboards/node_modules/sass-loader/dist/index.js:73:16)
           │              at Object.done [as callback] (/tmp/tmpydnnjqyv/OpenSearch-Dashboards/node_modules/neo-async/async.js:8067:18)
           │              at options.error (/tmp/tmpydnnjqyv/OpenSearch-Dashboards/node_modules/node-sass/lib/index.js:293:32)
           │           @ ./public/index.scss?v7dark 2:26-253
           │           @ ./public/index.scss
           │           @ ./public/index.ts
           │           @ /tmp/tmpydnnjqyv/OpenSearch-Dashboards/packages/osd-optimizer/target/worker/entry_point_creator.js
           │
           │          ERROR in ./public/index.scss?v8dark (/tmp/tmpydnnjqyv/OpenSearch-Dashboards/node_modules/css-loader/dist/cjs.js??ref--6-oneOf-2-1!/tmp/tmpydnnjqyv/OpenSearch-Dashboards/node_modules/postcss-loader/src??ref--6-oneOf-2-2!/tmp/tmpydnnjqyv/OpenSearch-Dashboards/node_modules/sass-loader/dist/cjs.js??ref--6-oneOf-2-3!./public/index.scss?v8dark)
           │          Module build failed (from /tmp/tmpydnnjqyv/OpenSearch-Dashboards/node_modules/sass-loader/dist/cjs.js):
           │          SassError: Top-level selectors may not contain the parent selector "&".
           │                  on line 1:10 of src/core/public/chrome/ui/header/_index.scss
           │                  from line 1:9 of src/core/public/chrome/ui/_index.scss
           │                  from line 1:9 of src/core/public/chrome/_index.scss
           │                  from line 5:9 of /tmp/tmpydnnjqyv/OpenSearch-Dashboards/src/core/public/index.scss
           │          >>   &.euiBody--headerIsFixed {
           │
           │             --^
           │
           │          SassError: SassError: Top-level selectors may not contain the parent selector "&".
           │                  on line 1:10 of src/core/public/chrome/ui/header/_index.scss
           │                  from line 1:9 of src/core/public/chrome/ui/_index.scss
           │                  from line 1:9 of src/core/public/chrome/_index.scss
           │                  from line 5:9 of /tmp/tmpydnnjqyv/OpenSearch-Dashboards/src/core/public/index.scss
           │          >>   &.euiBody--headerIsFixed {
           │
           │             --^
           │
           │              at Object.render [as callback] (/tmp/tmpydnnjqyv/OpenSearch-Dashboards/node_modules/sass-loader/dist/index.js:73:16)
           │              at Object.done [as callback] (/tmp/tmpydnnjqyv/OpenSearch-Dashboards/node_modules/neo-async/async.js:8067:18)
           │              at options.error (/tmp/tmpydnnjqyv/OpenSearch-Dashboards/node_modules/node-sass/lib/index.js:293:32)
           │           @ ./public/index.scss?v8dark 2:26-253
           │           @ ./public/index.scss
           │           @ ./public/index.ts
           │           @ /tmp/tmpydnnjqyv/OpenSearch-Dashboards/packages/osd-optimizer/target/worker/entry_point_creator.js
           │
           │          ERROR in ./public/index.scss?v7light (/tmp/tmpydnnjqyv/OpenSearch-Dashboards/node_modules/css-loader/dist/cjs.js??ref--6-oneOf-1-1!/tmp/tmpydnnjqyv/OpenSearch-Dashboards/node_modules/postcss-loader/src??ref--6-oneOf-1-2!/tmp/tmpydnnjqyv/OpenSearch-Dashboards/node_modules/sass-loader/dist/cjs.js??ref--6-oneOf-1-3!./public/index.scss?v7light)
           │          Module build failed (from /tmp/tmpydnnjqyv/OpenSearch-Dashboards/node_modules/sass-loader/dist/cjs.js):
           │          SassError: Top-level selectors may not contain the parent selector "&".
           │                  on line 1:10 of src/core/public/chrome/ui/header/_index.scss
           │                  from line 1:9 of src/core/public/chrome/ui/_index.scss
           │                  from line 1:9 of src/core/public/chrome/_index.scss
           │                  from line 5:9 of /tmp/tmpydnnjqyv/OpenSearch-Dashboards/src/core/public/index.scss
           │          >>   &.euiBody--headerIsFixed {
           │
           │             --^
           │
           │          SassError: SassError: Top-level selectors may not contain the parent selector "&".
           │                  on line 1:10 of src/core/public/chrome/ui/header/_index.scss
           │                  from line 1:9 of src/core/public/chrome/ui/_index.scss
           │                  from line 1:9 of src/core/public/chrome/_index.scss
           │                  from line 5:9 of /tmp/tmpydnnjqyv/OpenSearch-Dashboards/src/core/public/index.scss
           │          >>   &.euiBody--headerIsFixed {
           │
           │             --^
           │
           │              at Object.render [as callback] (/tmp/tmpydnnjqyv/OpenSearch-Dashboards/node_modules/sass-loader/dist/index.js:73:16)
           │              at Object.done [as callback] (/tmp/tmpydnnjqyv/OpenSearch-Dashboards/node_modules/neo-async/async.js:8067:18)
           │              at options.error (/tmp/tmpydnnjqyv/OpenSearch-Dashboards/node_modules/node-sass/lib/index.js:293:32)
           │           @ ./public/index.scss?v7light 2:26-254
           │           @ ./public/index.scss
           │           @ ./public/index.ts
           │           @ /tmp/tmpydnnjqyv/OpenSearch-Dashboards/packages/osd-optimizer/target/worker/entry_point_creator.js
           │
           │          ERROR in ./public/index.scss?v8light (/tmp/tmpydnnjqyv/OpenSearch-Dashboards/node_modules/css-loader/dist/cjs.js??ref--6-oneOf-3-1!/tmp/tmpydnnjqyv/OpenSearch-Dashboards/node_modules/postcss-loader/src??ref--6-oneOf-3-2!/tmp/tmpydnnjqyv/OpenSearch-Dashboards/node_modules/sass-loader/dist/cjs.js??ref--6-oneOf-3-3!./public/index.scss?v8light)
           │          Module build failed (from /tmp/tmpydnnjqyv/OpenSearch-Dashboards/node_modules/sass-loader/dist/cjs.js):
           │          SassError: Top-level selectors may not contain the parent selector "&".
           │                  on line 1:10 of src/core/public/chrome/ui/header/_index.scss
           │                  from line 1:9 of src/core/public/chrome/ui/_index.scss
           │                  from line 1:9 of src/core/public/chrome/_index.scss
           │                  from line 5:9 of /tmp/tmpydnnjqyv/OpenSearch-Dashboards/src/core/public/index.scss
           │          >>   &.euiBody--headerIsFixed {
           │
           │             --^
           │
           │          SassError: SassError: Top-level selectors may not contain the parent selector "&".
           │                  on line 1:10 of src/core/public/chrome/ui/header/_index.scss
           │                  from line 1:9 of src/core/public/chrome/ui/_index.scss
           │                  from line 1:9 of src/core/public/chrome/_index.scss
           │                  from line 5:9 of /tmp/tmpydnnjqyv/OpenSearch-Dashboards/src/core/public/index.scss
           │          >>   &.euiBody--headerIsFixed {
           │
           │             --^
           │
           │              at Object.render [as callback] (/tmp/tmpydnnjqyv/OpenSearch-Dashboards/node_modules/sass-loader/dist/index.js:73:16)
           │              at Object.done [as callback] (/tmp/tmpydnnjqyv/OpenSearch-Dashboards/node_modules/neo-async/async.js:8067:18)
           │              at options.error (/tmp/tmpydnnjqyv/OpenSearch-Dashboards/node_modules/node-sass/lib/index.js:293:32)
           │           @ ./public/index.scss?v8light 2:26-254
           │           @ ./public/index.scss
           │           @ ./public/index.ts
           │           @ /tmp/tmpydnnjqyv/OpenSearch-Dashboards/packages/osd-optimizer/target/worker/entry_point_creator.js
   │ERROR failure 2 min 53 sec
   │ERROR Error: webpack issue
   │          at createFailError (/tmp/tmpydnnjqyv/OpenSearch-Dashboards/packages/osd-dev-utils/target/run/fail.js:37:26)
   │          at Object.complete (/tmp/tmpydnnjqyv/OpenSearch-Dashboards/packages/osd-optimizer/src/optimizer/handle_optimizer_completion.ts:61:19)
   │          at TapSubscriber._complete (/tmp/tmpydnnjqyv/OpenSearch-Dashboards/node_modules/rxjs/src/internal/operators/tap.ts:140:25)
   │          at TapSubscriber.Subscriber.complete (/tmp/tmpydnnjqyv/OpenSearch-Dashboards/node_modules/rxjs/src/internal/Subscriber.ts:126:12)
   │          at MergeMapSubscriber.notifyComplete (/tmp/tmpydnnjqyv/OpenSearch-Dashboards/node_modules/rxjs/src/internal/operators/mergeMap.ts:171:24)
   │          at SimpleInnerSubscriber._complete (/tmp/tmpydnnjqyv/OpenSearch-Dashboards/node_modules/rxjs/src/internal/innerSubscribe.ts:39:17)
   │          at SimpleInnerSubscriber.complete (/tmp/tmpydnnjqyv/OpenSearch-Dashboards/node_modules/rxjs/src/internal/Subscriber.ts:126:12)
   │          at SafeSubscriber.subscriber.add.event$.subscribe [as _complete] (/tmp/tmpydnnjqyv/OpenSearch-Dashboards/packages/osd-optimizer/src/common/event_stream_helpers.ts:115:24)
   │          at SafeSubscriber.wrappedComplete (/tmp/tmpydnnjqyv/OpenSearch-Dashboards/node_modules/rxjs/src/internal/Subscriber.ts:248:54)
   │          at SafeSubscriber.__tryOrUnsub (/tmp/tmpydnnjqyv/OpenSearch-Dashboards/node_modules/rxjs/src/internal/Subscriber.ts:265:10)
   │          at SafeSubscriber.complete (/tmp/tmpydnnjqyv/OpenSearch-Dashboards/node_modules/rxjs/src/internal/Subscriber.ts:251:16)
   │          at Subscriber._complete (/tmp/tmpydnnjqyv/OpenSearch-Dashboards/node_modules/rxjs/src/internal/Subscriber.ts:148:22)
   │          at Subscriber.complete (/tmp/tmpydnnjqyv/OpenSearch-Dashboards/node_modules/rxjs/src/internal/Subscriber.ts:126:12)
   │          at MergeMapSubscriber.notifyComplete (/tmp/tmpydnnjqyv/OpenSearch-Dashboards/node_modules/rxjs/src/internal/operators/mergeMap.ts:171:24)
   │          at SimpleInnerSubscriber._complete (/tmp/tmpydnnjqyv/OpenSearch-Dashboards/node_modules/rxjs/src/internal/innerSubscribe.ts:39:17)
   │          at SimpleInnerSubscriber.Subscriber.complete (/tmp/tmpydnnjqyv/OpenSearch-Dashboards/node_modules/rxjs/src/internal/Subscriber.ts:126:12)

error Command failed with exit code 1.
info Visit https://yarnpkg.com/en/docs/cli/run for documentation about this command.
2021-12-06 19:52:00 ERROR    Error building OpenSearch-Dashboards, retry with: ./build.sh manifests/1.3.0/opensearch-dashboards-1.3.0.yml --component OpenSearch-Dashboards
Traceback (most recent call last):
  File "./src/run_build.py", line 83, in <module>
    sys.exit(main())
  File "./src/run_build.py", line 71, in main
    builder.build(build_recorder)
  File "/home/ec2-user/opensearch-build/src/build_workflow/builder_from_source.py", line 49, in build
    self.git_repo.execute(build_command)
  File "/home/ec2-user/opensearch-build/src/git/git_repository.py", line 83, in execute
    subprocess.check_call(command, cwd=cwd, shell=True)
  File "/usr/lib64/python3.7/subprocess.py", line 363, in check_call
    raise CalledProcessError(retcode, cmd)
subprocess.CalledProcessError: Command 'bash /home/ec2-user/opensearch-build/scripts/components/OpenSearch-Dashboards/build.sh -v 1.3.0 -p linux -a arm64 -s false -o builds' returned non-zero exit status 1.
```

**Comment by peterzhuamazon** [comment]
> You can also install chromium, e.g. in Fedora `dnf install chromium`

That is a workaround which would run but I am looking for a permanent solution so we dont have to install chromium manually every time.

Thanks.

**Comment by kavilla** [comment]
https://github.com/opensearch-project/OpenSearch-Dashboards/pull/1285 seeing issues not building on Linux x64 (although you can build on RPM for ARM on x64)

**Comment by bluesalt** [comment]
I guess I have figured out the root cause of the sass error issue.  
The libsass version used in this [#501](https://github.com/opensearch-project/OpenSearch-Dashboards/pull/501)  is **3.6.3** which is incompatible with the eui.  Thus the build will failed if libsass 3.6.3 is used.  
The tricky part of the story is that  the installation script of node-sass will try to grab a pre-compiled binary from github before it build the binary from source locally.  Because the versions specified in `package.json` are both `4.13.1`,   [the unreleased node-sass v5](https://github.com/sass/node-sass/tree/v5) imported in  [#501](https://github.com/opensearch-project/OpenSearch-Dashboards/pull/501) shares the same pre-built binary with version [node-sass 4.13.1](https://github.com/sass/node-sass/blob/v4.13.1) which actually uses libsass **3.5.3** and libsass 3.5.3 works happily with the eui.
In short, if the installation process failed to grap pre-built node-sass from github,  node-sass will be built from source. In that case, the build will fail since libsass 3.6.3 is used.   Either temporary network outage or non-existing pre-built binarie could make that happen. 
To fix that, we can bump node-sass version to a workable version such as 4.14.1.

**Comment by kristenTian** [comment]
Tried to build the package in a arm64 machine. Logging the following error during `yarn osd bootstrap`:
```
[centos@ip-172-31-6-108 OpenSearch-Dashboards]$ yarn osd bootstrap
yarn run v1.22.19
$ node scripts/osd bootstrap
 info [opensearch-dashboards] running yarn
 
$ node ./preinstall_check
[1/5] Validating package.json...
[2/5] Resolving packages...
warning Resolution field "typescript@4.0.2" is incompatible with requested version "typescript@~4.5.2"
[3/5] Fetching packages...
[4/5] Linking dependencies...
warning "workspace-aggregator-731a7ca1-da9a-4c1f-8dd4-30c0ecb6e564 > osd_tp_run_pipeline > @elastic/eui@1.0.0" has incorrect peer dependency "typescript@^4.0.5".
warning "@elastic/github-checks-reporter > @octokit/rest > @octokit/plugin-request-log@1.0.4" has unmet peer dependency "@octokit/core@>=3".
[5/5] Building fresh packages...
[1/18] ⡀ @elastic/eui
[6/18] ⡀ re2
[3/18] ⡀ core-js
[4/18] ⡀ fibers
error /home/centos/OpenSearch-Dashboards/node_modules/fibers: Command failed.
Exit code: 127
Command: node build.js || nodejs build.js
Arguments: 
Directory: /home/centos/OpenSearch-Dashboards/node_modules/fibers
Output:
gyp info it worked if it ends with ok
gyp info using node-gyp@7.1.2
gyp info using node@14.20.0 | linux | arm64
gyp info find Python using Python version 2.7.5 found at "/usr/bin/python"
gyp info spawn /usr/bin/python
gyp info spawn args [
gyp info spawn args   '/home/centos/OpenSearch-Dashboards/node_modules/node-gyp/gyp/gyp_main.py',
gyp info spawn args   'binding.gyp',
gyp info spawn args   '-f',
gyp info spawn args   'make',
gyp info spawn args   '-I',
gyp info spawn args   '/home/centos/OpenSearch-Dashboards/node_modules/fibers/build/config.gypi',
gyp info spawn args   '-I',
gyp info spawn args   '/home/centos/OpenSearch-Dashboards/node_modules/node-gyp/addon.gypi',
gyp info spawn args   '-I',
gyp info spawn args   '/home/centos/.cache/node-gyp/14.20.0/include/node/common.gypi',
gyp info spawn args   '-Dlibrary=shared_library',
gyp info spawn args   '-Dvisibility=default',
gyp info spawn args   '-Dnode_root_dir=/home/centos/.cache/node-gyp/14.20.0',
gyp info spawn args   '-Dnode_gyp_dir=/home/centos/OpenSearch-Dashboards/node_modules/node-gyp',
gyp info spawn args   '-Dnode_lib_file=/home/centos/.cache/node-gyp/14.20.0/<(target_arch)/node.lib',
gyp info spawn args   '-Dmodule_root_dir=/home/centos/OpenSearch-Dashboards/node_modules/fibers',
gyp info spawn args   '-Dnode_engine=v8',
gyp info spawn args   '--depth=.',
gyp info spawn args   '--no-parallel',
gyp info spawn args   '--generator-output',
gyp info spawn args   'build',
gyp info spawn args   '-Goutput_dir=.'
gyp info spawn args ]
gyp info spawn make
gyp info spawn args [ 'BUILDTYPE=Release', '-C', 'build' ]
make: g++: Command not found
make: *** [Release/obj.target/fibers/src/fibers.o] Error 127
gyp ERR! build error 
gyp ERR! stack Error: `make` failed with exit code: 2
gyp ERR! stack     at ChildProcess.onExit (/home/centos/OpenSearch-Dashboards/node_modules/node-gyp/lib/build.js:194:23)
gyp ERR! stack     at ChildProcess.emit (events.js:400:28)
gyp ERR! stack     at Process.ChildProcess._handle.onexit (internal/child_process.js:285:12)
gyp ERR! System Linux 4.18.0-348.20.1.el7.aarch64
gyp ERR! command "/home/centos/.nvm/versions/node/v14.20.0/bin/node" "/home/centos/OpenSearch-Dashboards/node_modules/.bin/node-gyp" "rebuild" "--release"
gyp ERR! cwd /home/centos/OpenSearch-Dashboards/node_modules/fibers
gyp ERR! node -v v14.20.0
gyp ERR! node-gyp -v v7.1.2
gyp ERR! not ok 
node-gyp exited with code: 1
Please make sure you are using a supported platform and node version. If you
would like to compile fibers on this machine please make sure you have setup your
build environment--
Windows + OS X instructions here: https://github.com/nodejs/node-gyp
Ubuntu users please run: `sudo apt-get install g++ build-essential`
RHEL users please run: `yum install gcc-c++` and `yum groupinstall 'Development Tools'` 
Alpine users please run: `sudo apk add python make g++`
/bin/sh: nodejs: command not found
make: Entering directory `/home/centos/OpenSearch-Dashboards/node_modules/fibers/build'
  CXX(target) Release/obj.target/fibers/src/fibers.o
 
 
 
 
ERROR [bootstrap] failed:
ERROR Error: Command failed with exit code 127: /home/centos/.nvm/versions/node/v14.20.0/lib/node_modules/yarn/bin/yarn.js install --non-interactive
          at makeError (/home/centos/OpenSearch-Dashboards/packages/osd-pm/dist/index.js:25150:11)
          at handlePromise (/home/centos/OpenSearch-Dashboards/packages/osd-pm/dist/index.js:24085:26)
          at processTicksAndRejections (internal/process/task_queues.js:95:5)
          at async installInDir (/home/centos/OpenSearch-Dashboards/packages/osd-pm/dist/index.js:23821:3)
          at async Project.installDependencies (/home/centos/OpenSearch-Dashboards/packages/osd-pm/dist/index.js:14780:5)
          at async Object.run (/home/centos/OpenSearch-Dashboards/packages/osd-pm/dist/index.js:9003:11)
          at async runCommand (/home/centos/OpenSearch-Dashboards/packages/osd-pm/dist/index.js:57870:5)
          at async Module.run (/home/centos/OpenSearch-Dashboards/packages/osd-pm/dist/index.js:263:3)
error Command failed with exit code 1.
info Visit https://yarnpkg.com/en/docs/cli/run for documentation about this command

**Comment by kristenTian** [comment]
https://github.com/opensearch-project/OpenSearch-Dashboards/pull/1113/files  percy/agent was removed here

**Comment by seraphjiang** [comment]
Thank you for reporting this issue. This issue has been open for a long time without recent activity, and OpenSearch Dashboards has since moved to the 3.x release line (currently 3.6, with 3.7 upcoming). The last supported 2.x release is 2.19.x.

We're closing this as part of a housekeeping effort to keep the backlog actionable. If this issue is still reproducible on a supported version (2.19.x or 3.x), please feel free to reopen this issue or file a new one with updated reproduction steps and version details.

We appreciate your contribution to OpenSearch!
